govec/govec.go

```diff
@@ -560,3 +576,21 @@ func (gv *GoLog) UnpackReceive(mesg string, buf []byte, unpack interface{}, opts
 	gv.mutex.Unlock()
 
 }
+
+//StartBroadcast is to be called just prior to starting a message broadcast
```

Comment:
Improve the doc and describe what `StartBroadcast` does actually. Current text only describes how it should be used. Also, emphasize that `StartBroadcast` must have a corresponding `StopBroadcast`, otherwise you'll end up in a deadlock.